winrm/client.go

```diff
@@ -18,10 +19,22 @@ type Client struct {
 	password  string
 	useHTTPS  bool
 	url       string
+	err       error
 	http      HttpPost
 	transport http.RoundTripper
 }
 
+// Map of known terminating errors, this will be utilized for returning error codes and error messages on
+// fatal errors
+var errorCode = map[string]int{
```

Comment:
Yes, I remember I asked @pecigonzalo to make the change. But you're right it might be more golang idiomatic the other way. I'll change that.